### PR TITLE
781 Intro Page

### PR DESCRIPTION
**Pull request by igreulich**
## Description
As a Veteran submitting a claim for the contention of PTSD on the Vets.gov platform, I need to be properly oriented so that I can easily transition into the guided flow for the Form0781/0781a from the 526EZ flow.

## Testing done
N/A

## Screenshots
See the initial page here - 
https://bahdigital.invisionapp.com/share/F6IA8LNDMXG#/screens

## Acceptance criteria
- [ ] The Veteran will be shown an explanation screen that introduces him/her to this workflow, per the prototype (https://bahdigital.invisionapp.com/share/F6IA8LNDMXG#/screens/295996741)
- [ ] The Veterans Crisis Line telephone number should be hyperlinked for the user and clicking on it opens a link to dial from your computer/mobile device
- [ ] The Veterans Crisis Line confidential online chat should be hyperlinked for the user and clicking on it opens a link to chat from your computer/mobile device
- [ ] The Veterans Crisis Line text message option should be hyperlinked for the user and clicking on it opens a link to text from your computer/mobile device

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/781/definitions/toursOfDuty.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
```
Comment (robfelder) [inline src/applications/disability-benefits/781/definitions/toursOfDuty.js]: Is this file necessary?

File: src/applications/disability-benefits/781/helpers.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export const introductionText = (
+  <div>
+    <p>
+      We'll now ask you questions about the stressful event or events related to your PTSD. We understand that some of the questions maybe difficult to answer. The information you provide here will help us understand you situation and research your claim.
```
Comment (robfelder) [inline src/applications/disability-benefits/781/helpers.js]: There's a misspelling here. "...help us understand you situation..." should be "...help us understand your situation..."

**Review by robfelder — CHANGES_REQUESTED** [review]
Outside of the misspelling and the one page in question, the code LGTM! The linter errors and unit tests have to be resolved before merging though.

**Comment by igreulich** [comment]
@robfelder Need a recheck, please.

**Comment by robfelder** [comment]
Looks like there are still some linter errors @igreulich

**Comment by igreulich** [comment]
@robfelder , I will resolve these conflicts, after I have gotten the Ack-spelling stuff pushed through.

**Comment by igreulich** [comment]
@robfelder You are up, again!

**Comment by robfelder** [comment]
@igreulich there are still some linter errors, but I'm building on top of this PR and have resolved them already. Unit test error is just like before. Imee's unit test should resolve that.

**Pull request merged**